packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,377 @@
+## User Types API
```

Comment:
before the title, we need to put in metadata:
```
---
title: User Types
category: management
---

Reply:
Added.